pegasus/sites.v3/hourofcode.com/public/survey/splat.haml

```diff
@@ -25,160 +25,8 @@ rightbar: blank
-      %label.control-label{for: "students-number-ethnicity"}
-        In total, how many of the students who participated were from historically underrepresented ethnicities in computer science:
-        African American, Hispanic/Latino/Latina, Pacific Islander, or Native American/Alaskan Native? Enter a whole number.
-        %span.form-required-field *
-      %div
-        %input#students-number-ethnicity.form-control{name: "students_number_ethnicity_s", placeholder: "", type: "text", required: true}/
-
-  .form-group
-    %label.control-label{for: "volunteer-participation"} Did a volunteer participate in the Hour of Code?
-    %span.form-required-field *
-    %div
-      %select#volunteer-participation.form-control{name: "volunteer_participation_s", type: "select", multiple: true, required: true}
-        %option{value: nil, selected: true}
-        - HocSurvey2015.volunteer_participation_values.each  do |label|
-          %option{value: label}= label
-
-  .form-group
-    %label.control-label{for: "teacher-how-heard"} How did you hear about the Hour of Code? (Choose all that apply)
-    %span.form-required-field *
-    %div
-      %select#teacher-how-heard.form-control{name: "teacher_how_heard_ss[]", type: "select", multiple: true, required: true}
-        %option{value: nil, selected: true}
-        -HocSurvey2015.teacher_how_heards.each do |label|
-          %option{value: label}= label
-
-    %div#teacher-how-heard-other-wrapper{style: 'display: none;'}
-      %label.control-label{for: "teacher-how-heard-other"}
-        %em{style: 'font-weight: normal;'} Other method
-      %div
-        %input#tacher-how-heard-other.form-control{name: "teacher_how_heard_other_s", placeholder: "", type: "text"}/
-
-  .form-group
-    %label.control-label{for: "hoc-centerpiece"}
-      Should we keep the Hour of Code as the centerpiece activity of Computer Science Education Week,
-      and keep “Hour of Code” as the unifying brand for all the other activities that educators organize throughout the week?
-    %span.form-required-field *
-    %div
-      %select#hoc-centerpiece.form-control{name: "hoc_centerpiece_s", type: "select", required: true}
-        %option{value: nil, selected: true}
-        - HocSurvey2015.hoc_centerpiece_values.each  do |label|
-          %option{value: label}= label
-
-  .form-group
-    %label.control-label{for: "event-improvement"} How do you think we could improve the Hour of Code? (optional, limit 250 words)
-    %div
-      %textarea#event-improvement.form-control{name: "event_improvement_s", placeholder: "", style: "height: 8em;"}
-
-  %input{name:'code_s', type:'hidden', value:code}
-
-  .form-group.submit-btn
-    %button#btn-submit.btn.btn-default{type: "submit"} Submit
-
-#thanks{:style=>"display: none;"}
-  :markdown
-    Thank you for completing the survey. If you selected a prize, you will receive an email with information about how to claim it.
+%div
```

Comment:
look again. I don't know what you're talking about...

and by that I mean thanks for pointing that out! removed